src/Altinn.Notifications.Email.Core/Configuration/TopicSettings.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace Altinn.Notifications.Email.Core.Configuration;
+
+/// <summary>
+/// Configuration object used to hold topic names for core services to publish to in Kafka.
+/// </summary>
+public class TopicSettings
+{
+    /// <summary>
+    /// The name of the email sending accepted topic
+    /// </summary>
+    /// Todo: Consider renaming, content on topic is operation id and notifications. e.g. NewOprationsTopic? 
+    public string EmailSendingAcceptedTopicName { get; set; } = string.Empty;
+
+    /// <summary>
+    /// The name of the email sending accepted retry topic
+    /// </summary>
+    /// Todo: Consider renaming, content on topic is operation id and notifications. e.g. NewOprationsTopic? 
+    public string EmailSendingAcceptedRetryTopicName { get; set; } = string.Empty;
+    
+    /// <summary>
+    /// The name of the email operation result topic
+    /// </summary>    
+    public string EmailOperationResultTopicName { get; set; } = string.Empty;
```

Comment:
If we try to name topics to hold information about events, the names should reflect the event. One example could be `EmailSendingFinished` or something.

Reply:
new topic names agreed on and implemented